docs/tutorials/Containers.md

```diff
@@ -178,27 +195,48 @@ To run Singularity on a job scheduler, the singularity command needs to be passe
 For example, in SLURM, we can use the normal SLURM configuration as explained in the [SLURM documentation](../backends/SLURM), however we'll add a `submit-docker` block to execute when a task is tagged with a docker container. 
 
 When constructing this block, there are a few things to keep in mind:
-- Make sure Singularity is loaded (and in our path), for example you can call `module load Singularity/3.0.1`.
-- We should treat worker nodes as if they do not have stable access to the internet or build access, so we will pull the container before execution of the task.
-- It's a good idea to ask Singularity to build the image into the execution directory of the task as an artifact, and to save rendering time on the worker node.
-- We'll use the docker name (potentially including the docker hash) to generate a compliant filename that can be looked up later.
-- If the container exists, there is no need to rebuild the container.
+- Make sure Singularity is loaded (and in our path), for example you can 
+  call `module load Singularity/3.0.1` if module is installed on your system.
```

Comment:
This version is fairly old - any reason to use it for the example?

Reply:
I did not put it there. This was already there and I did not change it. It may be a good thing to remove things that reference version as these get outdated eventually.

To be frank. I wonder why the `module load` singularity is there in the first place as you can also use the path to the singularity executable. For example `/clusterfs/shared-software/singularity/3.5.3/bin/singularity`.

Reply:
The module load sometimes also sets SINGULARITY_CACHEDIR or other envars.

Reply:
While we're here, we might as well bump the singularity version to something newer but still stable.

Reply:
My thought when we wrote it was, people are using slurm which means there's a good chance they're using or are familiar with modules.

Reply:
`module load Singularity` without a version should default to the latest version. So the version statement can be dropped altogether.